### ISSUE TITLE
Failed to read package metadata 

### ISSUE THREAD

**Issue by jasonf20** [SymbolSource/SymbolSource.Community#24]
Hi,  i have hosted a clean project with Symbolsource. Basic. Server installed.  When I run the diagnostics i get. Failed to read package metadata: cannot access a closed file. this happened in the diagnosis test, and from NuGet push. 

Is there something I am doing wrong or is this a bug (seems like a bug to me)?

**Comment by Elephant-Vessel** [comment]
Yes, is this project still alive? The old version of nuget breaks with all packages that use the developmentDependency attribute in the nuspec metadata configurations.

**Comment by amamh** [comment]
This project should be avoided, it's a major waste of time. I'm giving up on it, hopefully there is a good alternative.

**Comment by jmecosta** [comment]
Proget?

**Comment by amamh** [comment]
@jmecosta didn't know about that.. are you using it?

I'm now in the middle of moving to Gitlink.

**Comment by jmecosta** [comment]
yep, we use in production and its a very solid product. and its free

**Comment by amamh** [comment]
@jmecosta  man, I've just started using it.. it's fucking great. I use it now for both NuGet packages and as a symbol source. It took me like 15 minutes total. I spent already almost a whole week fiddling around with the horrible community NuGet server and this crappy SymbolSource project. I was just about to waste another day trying to get Gitlink to work with my internal Stash server.

Thank God you told me about Proget.
